### DISCUSSION TITLE
Allow freight stations to automatically transfer goods from warehouses to businesses.

### DISCUSSION THREAD

**Discussion by murolem** [voxeltycoon/issues#317] [General] [answered]
Currently, if goods are being loaded to the warehouse in any other way than through freight station, they are NOT going to be transferred to the businesses in range, which is kinda silly.

The solution for now is to create a route like this (which is just weird):
1. Load at freight station.
2. Unload at freight station.
3. (optional) waypoint somewhere near (to not clog the traffic).

So if this implemented, it will make this behavior obsolete and replace it with more obvious behavior instead.

(this issue currently makes up like 20% of all the discussion in discord server)

**Comment by DBT85** [comment]
My preferred solution, have train/truck stations only deliver to warehouses within their AOE, and truck stations deliver directly to business (or another station with more adjoined warehouses).

Currently if a business springs up 1 tile outside of a stations AOE, you have to build another to capture it (and still have that new station in range of a WH) and then either cheese it like you mention or actually deliver by truck from one station to another. Both feel silly.

A freight stations AOE would only capture warehouses and a truck station could deliver to any business anywhere on the map with the obvious consequences of having too many trucks visiting one station being problematic and eliminating this daft cheese.

**Comment by IDragonfire** [comment]
My suggestion: https://github.com/voxeltycoon/suggestions/discussions/394

**Comment by katepolak** [comment, marked as answer]
This is a duplicate suggestion of #475, please read the existing idea for how this could work there and if you feel that yours has benefits over those feel free to post it there.
